--- dynamite-0.1/configure.ac.in	2003-08-22 11:55:42.000000000 +0300
+++ dynamite-0.1-pc/configure.ac.in	2007-11-12 02:38:07.246465852 +0200
@@ -32,7 +33,7 @@
 dnl Checks for typedefs, structures, and compiler characteristics.
 AC_C_CONST
 
-AC_OUTPUT([Makefile
+AC_OUTPUT([libdynamite.pc
+           Makefile
            lib/Makefile
            src/Makefile])
-
--- dynamite-0.1/Makefile.am~	2003-08-22 21:06:41.000000000 +0300
+++ dynamite-0.1/Makefile.am	2007-11-12 02:40:59.569133658 +0200
@@ -7,6 +7,9 @@
 m4datadir = $(datadir)/aclocal
 m4data_DATA = dynamite.m4
 
+pkgconfigdir = $(libdir)/pkgconfig
+pkgconfig_DATA = libdynamite.pc
+
 EXTRA_DIST = $(m4data_DATA)
 
 SUBDIRS = lib src .
--- /dev/null	2006-03-28 14:00:37.000000000 +0300
+++ dynamite-0.1/libdynamite.pc.in	2007-11-12 02:38:07.236465000 +0200
@@ -0,0 +1,10 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: libdynamite
+Description: PKWARE Data Compression decompressor
+Version: @VERSION@
+Libs: -L${libdir} -ldynamite
+Cflags: -I${includedir}
